src/jit/compiler.h

```diff
@@ -7169,6 +7169,17 @@ public :
         inline bool         IsReadyToRun() { return false; }
 #endif
 
+        // true if we must generate compatible code with Jit64 quirks
+        inline bool         IsJit64Compat()
+        {
+#if defined(_TARGET_AMD64_) && !defined(FEATURE_CORECLR)
+            // JIT64 interop not required for ReadyToRun since it can simply fall-back
+            return !(*this).IsReadyToRun();
```

Comment:
`(*this).` looks pretty unusual here - it is not used anywhere else in the JIT codebase. Is it really needed?